Src/Common/CommonEventSource.cs

```diff
@@ -0,0 +1,71 @@
+﻿namespace Microsoft.ApplicationInsights.Common
+{
+    using System;
+#if NET45
+    using System.Diagnostics.Tracing;
+#endif
+    using System.Globalization;
+    using Microsoft.ApplicationInsights.Extensibility.Implementation.Tracing;
+#if NET40
+    using Microsoft.Diagnostics.Tracing;
+#endif
+
+    /// <summary>
+    /// ETW EventSource tracing class.
+    /// </summary>
+    [EventSource(Name = "Microsoft-ApplicationInsights-Extensibility-Common")]
```

Comment:
`Common` is a bad name for eventsource. Can you name is cross components correlation instead